tests/fileio/fileiotests.fz

```diff
@@ -0,0 +1,79 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of tests for File I/O operations
+#
+#  Author: Wael Youssfi (wael.youssfi@tokiwa.software)
+#
+# -----------------------------------------------------------------------
+
+fileiotests : fuzion.std.fileio =>
+
+  dir := "testdir"
+  newdir := "newdir"
+  file := dir + "/" + "testfile"
+
+  content := "Hello world 🌍"
+
+  f := io.file
+
+  writer := f.write
+  reader := f.read
+  stats := f.stat
+
+  say "$dir exists: {exists dir}"
+  match writer.create_dir dir
+    _ unit => say "$dir was created"
```

Comment:
the `match` syntax allows to just not give a variable name if you do not need it, so `_` can be removed.